<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 63, 80)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageHome">
       <widget class="QPushButton" name="btnQuizSubjects">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>200</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Quiz Subjects</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLogout">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>310</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>LogOut</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>QUIZ APP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageQuizSubjects">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>SELECT SUBJECT FORADDING QUIZ QUESTIONS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBiology">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>130</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Biology</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnComputing">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>190</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Computing</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCustom">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnGeography">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>310</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Geography</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBack">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>450</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageAddQuiz">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ADD QUESTIONS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBack_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>480</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtQuestion">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>481</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>122</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Question</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>232</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Correct Answer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtCorrectAnswer">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>270</y>
          <width>481</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>561</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnAddQuestion">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>350</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Add Question</string>
         </property>
        </widget>
       </widget>
       <zorder>frame</zorder>
       <zorder>label_4</zorder>
       <zorder>btnBack_2</zorder>
       <zorder>txtQuestion</zorder>
       <zorder>label_5</zorder>
       <zorder>label_6</zorder>
       <zorder>txtCorrectAnswer</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
